dist/ruff.toml

```
@@ -23,6 +19,7 @@ ignore = [
     "F405", # undefined-local-with-import-star-usage
     "F504", # percent-format-extra-named-arguments
     "F523", # string-dot-format-extra-positional-arguments
+    # Ignore to allow flexibility in code formatting.
     "F541", # f-string-missing-placeholders
```

Can we put this on the same line as the rule being ignored? It looks like we're also ignoring F811, F821, F841 for reasons of flexibility.
```suggestion
    "F541", # f-string-missing-placeholders. Ignored to allow flexibility in code formatting.
```